script.js

```diff
@@ -1,165 +1,165 @@
 // create library and constructor
-let library = []
-const $bookList = document.querySelector("ul")
-const $addBookButton = document.querySelector(".add-book-button")
-function book(title, author, pages, read) {
-    this.title = title;
-    this.author = author;
-    this.pages = pages;
-    this.read = read;
+const library = [];
+const $bookList = document.querySelector('ul');
+const $addBookButton = document.querySelector('.add-book-button');
+function Book(title, author, pages, read) {
+  this.title = title;
+  this.author = author;
+  this.pages = pages;
+  this.read = read;
```

Comment:
- [ ] REQUIRED This function is still using the `function` keyword instead of arrow functions syntax.